concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -110,6 +110,7 @@ datadog_api_key: ${datadog_api_key:-}
 datadog_app_key: ${datadog_app_key:-}
 enable_datadog: ${ENABLE_DATADOG}
 enable_cve_notifier: ${ENABLE_CVE_NOTIFIER:-false}
+enable_cve_monitor: ${ENABLE_CVE_MONITOR:-false}
```

Comment:
As I see more and more of these settings, more I think that we should split the config as [commented here](https://github.com/alphagov/paas-cf/pull/584#issuecomment-258907304). 

We should have one place (e.g. .yaml per environment) where we enable or disable this, and that can be read consistently across the pipeline, instead of explicitly passing it each time.

But not an issue to solve in this PR.